.circleci/config.yml

```diff
@@ -134,7 +134,7 @@ commands:
       - run:
           name: "Run Migrations"
           command: |
-            cf run-task "tariff-<< parameters.service >>-backend-worker-<< parameters.environment_key >>" --command "rake db:migrate" --name "db-migrate"
+            cf run-task "tariff-<< parameters.service >>-backend-worker-<< parameters.environment_key >>" --command "cd app && rake db:migrate" --name "db-migrate" --wait
```

Comment:
The `cd app` part isn't right solution - the underlying issue is the Dockerfile should be changed to put the app in the correct location (`/app` instead of `/app/app`)

I'd suggest remove the cd part, push to the PR - we can then verify it fails to deploy (because migrations step _should_ fail at this point). This is the important bit because it means we know the migrations step will error in CI if developers add invalid migrations by mistake in the future.

Once we know the PR's CI run is failing in the manner we expect (its failing on the migrations step) - then add the missing `/` in the Dockerfile to 'fix' the PR?

Reply:
Yes, this commit was just a test, after suspecting a further commit targeting the Dockerfile  caused migrations to be out of sync. 
I probably should move this PR to draft again until this is sorted.